invenio_app_rdm/theme/static/robots.txt

```diff
@@ -2,4 +2,11 @@ User-agent: *
 Disallow: /search
 Disallow: /api
 Disallow: /administration
+Disallow: /account
+Disallow: /uploads
+Disallow: /me
+Disallow: /login
+Disallow: /logout
+Disallow: /records/*/files
+Disallow: /records/*/preview
```

Comment:
These are probably the most relevant routes here, also for unauthenticated guests.

Reply:
Google datasets actually does also index some [files for preview](https://datasetsearch.research.google.com/search?src=3&query=canada%20water%20quality&docid=L2cvMTFqbnoyanRwNA%3D%3D) (e.g. CSVs). I'll open a PR to remove this line specifically.